features/gebruik_in_lopende_tekst.feature

```diff
@@ -34,138 +34,126 @@ Rule: gebruikInLopendeTekst voor een persoon zonder adellijke titel of predicaat
   - Aanduiding naamgebruik "E" (eigen naam) wordt gehanteerd voor een persoon die geen actuele en geen ontbonden huwelijken/partnerschappen heeft gehad
```

Comment:
Waarom wordt deze regel opgenomen? Is het niet zo dat in dit geval 'aanduiding naamgebruik (61.10)' al de waarde 'E' zal hebben of kan het voorkomen dat deze niet gevuld is of toch een andere waarde heeft?

Als dat laatste het geval is vraag ik me het volgende af:
* moet de GBA API, als fields de waarde 'adressering.gebruikInLopendeTekst' heeft, niet ook een lege partner array meesturen zodat de proxy deze conclusie kan trekken?
* wat als er toch partners gevonden worden zonder dat 'aanduiding naamgebruik (61.10)' een waarde heeft?

Reply:
Ik denk dat ik inmiddels begrijp wat hier bedoeld wordt.
Als een persoon geen actuele of ontbonden huwelijken/partnerschappen heeft gehad dan wordt, ongeacht de daadwerkelijke waarde, voor 'aanduiding naamgebruik (61.10)' de waarde 'E' gehanteerd. Lijkt me dat bullet 1 in dat geval idd van toepassing is.